extensions/linear/src/api/getTeams.ts

```diff
@@ -50,6 +43,7 @@ export async function getTeams() {
         }
       }
     `,
```

Comment:
Few more questions about this query:

```
activeCycle {
              id
              number
            }
```
Do you need an array of all active cycles in this response? Or you just want to know whether cycles are enabled for a given team? If the latter, there's a really quick way to do it using `team { cyclesEnabled }`

Reply:
```
organization {
   logoUrl
}
```

Queries organization here for each team. If you really need it in this query, then we need to move it outside of team query

```
organization {
   logoUrl
}
teams {
   nodes {
   }
}
```

Reply:
> Do you need an array of all active cycles in this response?

@suevalov We look for the active cycle for the team and then use that id to get all the issues, I think `cyclesEnabled` might not fit the second criteria. Please let us know if we can leverage some other method to get this info.

https://github.com/raycast/extensions/blob/3d1cdb073466978cdbec889db92e49b9611c0827/extensions/linear/src/active-cycle.tsx#L25-L31

> If you really need it in this query, then we need to move it outside of team query

Moved it outside the team query, but we just use the org logo as a fallback in case team logo is not defined.

https://github.com/raycast/extensions/blob/35dd2d5ef0e93e6db72bb92974da0146ffb67a13/extensions/linear/src/helpers/teams.ts#L6